### ISSUE TITLE
Using `TemplateExporter.exclude_output_prompt` will remove output

### ISSUE THREAD

**Issue by AKuederle** [jupyter/nbconvert#939]
When the option `TemplateExporter.exclude_output_prompt` is used in combiantion with the latex export will remove all non plain text output and not just the output prompt

For example, this cell will produce no output under the described conditions
```python
import pandas as pd
iris = pd.read_csv('https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv')
iris.head()
```

**Comment by t-makaro** [comment]
What version of nbconvert are you using? I believe this was fixed in 5.4.0 by #734

**Comment by t-makaro** [comment]
Closing as I cannot reproduce on master. Please comment to reopen if this is still an issue,
